src/screens/SplashScreen/index.js

```diff
@@ -5,7 +5,8 @@ import {Linking} from 'react-native';
 import {StackActions} from '@react-navigation/native';
 import {useNavigation} from '@react-navigation/core';
 
-import useProfileHook from '../../hooks/core/profile/useProfileHook';
+import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
+import TokenStorage, {ITokenEnum} from '../../utils/storage/custom/tokenStorage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useUserAuthHook` and `TokenStorage` seems fine. However, ensure that the tokens stored in `TokenStorage` are securely managed and not exposed to any potential security risks.
```diff
- import useProfileHook from '../../hooks/core/profile/useProfileHook';
+ import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
+ import TokenStorage, {ITokenEnum} from '../../utils/storage/custom/tokenStorage';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->